### ISSUE TITLE
TypeError: OAuthStrategy requires a consumerKey option

### ISSUE THREAD

**Issue by Orbiter** [loklak/loklak_webclient#87]
I did npm install in / and /server and started gulp dev in /
That resulted in
```
/Users/admin/git/loklak_webclient/server/node_modules/passport-twitter/node_modules/passport-oauth1/lib/strategy.js:83
  if (!options.consumerKey) { throw new TypeError('OAuthStrategy requires a co
                                    ^
TypeError: OAuthStrategy requires a consumerKey option
    at Strategy.OAuthStrategy (/Users/admin/git/loklak_webclient/server/node_modules/passport-twitter/node_modules/passport-oauth1/lib/strategy.js:83:37)
    at new Strategy (/Users/admin/git/loklak_webclient/server/node_modules/passport-twitter/lib/strategy.js:53:17)
    at Object.<anonymous> (/Users/admin/git/loklak_webclient/server/index.js:37:14)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Function.Module.runMain (module.js:501:10)
    at startup (node.js:129:16)
    at node.js:814:3
[09:29:28] [nodemon] app crashed - waiting for file changes before starting...
[BS] Proxying: http://localhost:3001
[BS] Now you can access your site through the following addresses:
[BS] Local URL: http://localhost:3000
[BS] External URL: http://192.168.1.31:3000
```

**Comment by julianfox** [comment]
You have to create a Twitter app at https://apps.twitter.com

Set the twitterConsumerKey var in custom_configFile.json to set the Consumer Key (API Key) from your Twitter app
Set the twitterConsumerSecret var in custom_configFile.json to set the Consumer Secret from your Twitter app

I updated the Readme about it

**Comment by sudheesh001** [comment]
Fixed via 54dd21744ccf43049d8a78ebee8e6c734eabc170

**Comment by Orbiter** [comment]
This is not the complete solution. I expect that parts of the application, which do not use or need a twitter app key should work without the configuration of a consumer key+secret
